console/src/utils/migration/utils.ts

```diff
@@ -349,13 +349,11 @@ export const getDownQueryComments = (
   source: string
 ) => {
   if (Array.isArray(upqueries) && upqueries.length >= 0) {
-    let comment = `-- Could not auto-generate a down migration.
--- Please write an appropriate down migration for the SQL below:`;
-    comment = upqueries.reduce(
-      (acc, i) => `${acc}
--- ${i.args.sql}`,
-      comment
-    );
+    const comment = [
+      "Could not auto-generate a down migration.",
+      "Please write an appropriate down migration for the SQL below:",
+      ...upqueries.map((i) => i.args.sql)
+    ].join("\n").replace(/\r?\n/g, "$1-- ");
```

Comment:
could you try this one?
small changes on the regex to make sure all carrier returns and line breaks are captured + commented out the first line directly.
```suggestion
    const comment = [
      '-- Could not auto-generate a down migration.',
      'Please write an appropriate down migration for the SQL below:',
      ...upqueries.map(i => i.args.sql),
    ]
      .join('\n')
      .replace(/(\r\n)+|\r+|\n+/g, '\n-- ');
```

Reply:
with the above suggestion, I was able to generate the following

```sql
-- Could not auto-generate a down migration.
-- Please write an appropriate down migration for the SQL below:
-- CREATE OR REPLACE VIEW author_publics_2 AS
--   SELECT id, name,city, email
--     FROM author;
```

Reply:
Bare carriage returns aren't used in the modern world, it's either `\n` or `\r\n`. I've revised and rebased the PR to ensure the first line is commented, newlines are normalized, lines do not have trailing spaces, and the file ends with a newline.